src/package_index.rs

```diff
@@ -184,6 +223,32 @@ impl PackageIndex {
         }
     }
 
+    /// Read the list of owners for a scope from the index
+    pub fn get_scope_owners(&self, scope: &str) -> anyhow::Result<Option<Vec<u64>>> {
+        let mut path = self.path.clone();
+        path.push(scope);
+        path.push("owners.json");
+
+        let file = match File::open(path) {
+            Ok(file) => Ok(Some(file)),
+            Err(error) => match error.kind() {
+                ErrorKind::NotFound => Ok(None),
```

Comment:
You can just return `Ok(Vec::new())` here and simplify a lot of this code. Then, `file` can just be a `File`.